plugins/ctf/fs-sink/write.c

```diff
@@ -315,50 +295,37 @@ struct fs_writer *insert_new_writer(
 	int nr_stream, i;
 
 	if (writer_component->single_trace && writer_component->nr_traces > 0) {
-		fprintf(writer_component->err, "[error] Trying to process more "
-				"than one trace but --single-trace mode "
-				"enabled\n");
+		BT_LOGE_STR("Trying to process more than one trace but --single-trace mode enabled");
```

Comment:
`--single-trace` does not make sense from this plugin's point of view; it looks like a CLI option, and this option does not event exist. _but single trace mode enabled_ would be enough.